packages/e2e-tests/specs/editor/various/adding-blocks.test.js

```diff
@@ -276,10 +276,12 @@ describe( 'adding blocks', () => {
 		await insertBlock( 'Paragraph' );
 		await page.keyboard.type( 'First paragraph' );
 		await insertBlock( 'Image' );
+		await showBlockToolbar();
```

Comment:
Why do you think this is needed?

Reply:
I did not spend much time inspecting the failure, but I noticed that when I used PUPPETEER_SLOWMO=80 the block toolbars appeared and the tests passed, when I used default speed the toolbars sometimes appeared sometimes not appeared, and the tests not always passed locally. So forcing toolbar the toolbars to appeared seemed like a plausible fix.